website/content/docs/api-gateway/configuration/index.mdx

```diff
@@ -13,11 +13,12 @@ This topic provides an overview of the configuration items that enable Consul AP
 - [GatewayClass](/consul/docs/api-gateway/configuration/gatewayclass) defines a class of gateway resources that you can use as a template for creating gateways.
 - [GatewayClassConfig](/consul/docs/api-gateway/configuration/gatewayclassconfig) describes additional Consul API Gateway-related configuration parameters for the GatewayClass resource.
 - [Routes](/consul/docs/api-gateway/configuration/routes) specifies the path from the gateway to the backend service(s) client to the listener.
+- [Intentions](/consul/docs/api-gateway/configuration/intentions) specifies traffic communication rules between Consul API Gateway and services in the Consul service mesh.
```

Comment:
```suggestion
- [Intentions](/consul/docs/connect/config-entries/service-intentions) specifies traffic communication rules between services in the mesh. You can also use them specify rules for communication between services outside the mesh that send traffic through Consul API gateway to services in the service mesh.
```
I made another comment directly on the proposed SI conf page, but just be clear, we only want one source of truth for configuration reference. In this case, it should be the existing SI conf entry ref.

Reply:
That's fair - I was torn between putting the existing link there and creating a new one. 

I'd like to include the intentions example somewhere, maybe in the upgrade docs, since this is a new workflow with 1.16. What do you think @trujillo-adam ?

Reply:
The convention we started to implement is to add examples for specific use cases to the configuration reference page. https://developer.hashicorp.com/consul/docs/connect/config-entries/service-intentions#examples. The goal is to keep them all in one place as, well, a reference, and link to them as necessary. 
I wouldn't add the example to the upgrade docs, but it is something different that a user has consider when upgrading to 1.16. To that end we could mention that you need to configure intentions in those instructions and link to the example on the reference page.